tests/lib.rs

```diff
@@ -56,7 +63,7 @@ fn test_write_measurement() {
     rt.block_on(client.write_one(measurement, None).then(move |_| {
         client.query("select * from \"sut\"".to_string(), None)
     }).map(|res| {
-        let fixture = "{\"results\":[{\"series\":[{\"name\":\"sut\",\"columns\":[\"time\",\"boolean\",\"float\",\"integer\",\"string\",\"tag\",\"tag, with comma\",\"with, comma\"],\"values\":[[\"2015-06-11T20:46:02Z\",false,10,10,\"string\",\"value\",\"three, four\",\"comma, with\"]]}]}]}";
+        let fixture = "{\"results\":[{\"statement_id\":0,\"series\":[{\"name\":\"sut\",\"columns\":[\"time\",\"boolean\",\"float\",\"integer\",\"string\",\"tag\",\"tag, with comma\",\"with, comma\"],\"values\":[[\"2015-06-11T20:46:02Z\",false,10,10,\"string\",\"value\",\"three, four\",\"comma, with\"]]}]}]}\n";
```

Comment:
This is returned from the test query with Influx version `1.7.6`. How can the tests be futureproofed for all versions, with some returning the `statement_id` key and newline at the end.

Reply:
This should be updated: https://github.com/gobwas/influent.rs/blob/master/.travis.yml#L6-L7.

All tests pass with the modification above at version `1.7.6`.